js/models/conversations.js

```diff
@@ -395,6 +397,24 @@
         return this.get('type') === 'private';
     },
 
+    muteConversation: function() {
+        if (this.get('muted')) {
+            this.save({'muted': false});
+        } else {
+            this.save({'muted': true});
+        }
+    },
+    getMutedState: function() {
+      return this.get('muted');
+    },
+    getMutedIcon: function() {
+        if (this.getMutedState()) {
+            return { url: '/images/do_not_disturb.svg' };
```

Comment:
this background image should be constant, defined in css, then conditionally render the entire icon or not.